### DISCUSSION TITLE
Missing 'jsonschema' package in PyAEDT dependencies?

### DISCUSSION THREAD

**Discussion by Porta-L** [ansys/pyaedt#4214] [Q&A] [answered]
My projects were running correctly under PyAEDT 0.7.7.
When updating PyAEDT to the latest version 0.7.12, I encountered a problem related to the absence of the 'jsonschema' package when initializing a Maxwell3d project with the 'Maxwel3d()' command, which returned the following error: No module named 'jsonschema'.
This then works correctly once the module is installed, but shouldn't this module come with PyAEDT dependencies?

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @Porta-L ,

It is included in the main dependencies:

https://github.com/ansys/pyaedt/blob/main/pyproject.toml

please try to create a new virtual environment.
